### DISCUSSION TITLE
How to import/export posts as well as models?

### DISCUSSION THREAD

**Discussion by esilverman682** [wpengine/atlas-content-modeler#584] [Q&A / Help / Support] [answered]
Im trying to import export content located within the newly created model.  Example: I created a Model named "Products" and added child titles and content such as "business loans" and "equipment financing". When importing or exporting to a new wordpress instance from the "Tools" section it only shows the parent model. Is there a way to migrate the child content as well to lets say a new wordpress environment without losing all the added child data.
 
 
<img width="452" alt="Screen Shot 2022-07-03 at 12 13 36 PM" src="https://user-images.githubusercontent.com/76890619/177048110-6e089c89-716c-44a6-a3a1-89af7baf7d51.png">
<img width="1547" alt="Screen Shot 2022-07-03 at 12 12 51 PM" src="https://user-images.githubusercontent.com/76890619/177048111-919dba84-a75c-4d6c-9276-a9018dcc8ae3.png">

**Comment by nickcernis** [comment, marked as answer]
Hi, @esilverman682, and thanks for your question.

The Content Modeler → Tools page only exports the models themselves and not the posts/entries, as you found.

ACM offers a way to export models and entries (as well as other WP data) if you're comfortable using [WP-CLI](https://wp-cli.org/) — it's part of the ACM “Blueprints” feature:

- ACM Blueprints overview: https://github.com/wpengine/atlas-content-modeler/blob/main/docs/blueprints/index.md
- ACM import/export WP-CLI commands: https://github.com/wpengine/atlas-content-modeler/blob/main/docs/wp-cli/index.md

We hope to extend the Tools page to export entries too at some point for people who don't want to use WP-CLI, but I don't yet have a timeline for this.
